installer/src/systemd_boot/plan.rs

```diff
@@ -433,7 +433,7 @@ mod tests {
             can_touch_efi_vars: false,
             bootctl: Some(PathBuf::from("bootctl")),
             unified_efi: false,
-            signing_info,
+            signing_info: signing_info.into(),
```

Comment:
I had hoped that implementing `Deref` for `OptionalSigningInfo` would mean we would not need to edit any other code. Suggestions welcomed.